### PR TITLE
docs: HW seqs minor

### PR DESCRIPTION
**Pull request by bguiz**
**Description**:

## What

- root
  - change links to absolute url
- create-fund-account
  - update text to state "HEX-encoded" in example
  - change emoji check marks to markdown rendered checkmarks
  - add what you learnt checkmarks list under completed
  - use backticks for some prereqs
  - add git clone with HTTPS, and move SSH into expandable
- hfs-files
  - change emoji check marks to markdown rendered checkmarks
  - add what you learnt checkmarks list under completed
- hscs-smart-contract
  - draw attention to file names for solidity file vs script
  - update image for arkhia project details to highlight testnet selection drop down
  - reorder checks for `myContractExplorerUrl`
  - change emoji check marks to markdown rendered checkmarks
  - add what you learnt checkmarks list under completed
  - replace moniker with nickname
- hts-fungible-token
  - update to correct dir name within shell command
  - detailed explanation of token create transaction parameters added to collapsible area
  - fix a typo, 2x
  - change emoji check marks to markdown rendered checkmarks
  - add what you learnt checkmarks list under completed
  - replace moniker with nickname

## Why

- review from @a-ridley 
- review from @michielmulders

**Related issue(s)**:

- Preview PR: https://github.com/hedera-dev/preview-hedera-docs/pull/8
- Parent: https://github.com/hashgraph/hedera-docs/issues/485

**Notes for reviewer**:

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)
  - `npm run test`

### CODE REVIEW

File: tutorials/hello-world/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ To start, you **must** do the first sequence,
 as you will need a **funded account** to do any other tasks on Hedera.
 Subsequently, you can do the remaining sequences in any order.
 
-- [Create and fund an account](./create-fund-account/)
-- [HFS: Files](./hfs-files/)
-- [HTS: Fungible token](./hts-fungible-token/)
-- [HSCS: Smart contract](./hscs-smart-contract/)
+- [Create and fund an account](https://docs.hedera.com/hedera/tutorials/hello-world/create-fund-account)
```
Comment (theekrystallee) [inline tutorials/hello-world/README.md]: This is a sequence, so an ordered list makes more sense than unordered/bulletpoints.

**Review by theekrystallee — CHANGES_REQUESTED** [review]
There are resolve conflicts that need to be addressed. Please resubmit after resolving. Thanks

File: tutorials/hello-world/README.md (hunk 1 above)
Comment (bguiz) [reply tutorials/hello-world/README.md]: No, actually this is not a sequence, see the line immediately above:

> Subsequently, you can do the remaining sequences in any order.

**Comment by bguiz** [comment]
Merge conflicts resolved in https://github.com/hashgraph/hedera-docs/pull/507/commits/7b4f27ff3549e7f84aef43dbd7d99fe774f1a2dc
> There are resolve conflicts that need to be addressed. Please resubmit after resolving. Thanks

**Review by theekrystallee — APPROVED** [review]

**Review by theekrystallee — APPROVED** [review]
LGTM

**Pull request merged**